Formula/elasticsearch.rb

```diff
@@ -113,11 +128,18 @@ def plist; <<-EOS.undent
   end
 
   test do
-    system "#{libexec}/bin/plugin", "list"
+    if FileTest.exist?("#{libexec}/bin/plugin")
```

Comment:
You can use `devel` and `head` as the modifier down here as well if you want to, they just become `if head?` and `if devel?` in `test do` rather than the `if build.head?` style equivalents.